### PR TITLE
Code Editor Line Numbers

### PR DESCRIPTION
**Pull request by pickjasmine**
Adds line numbers to the text area where you write your Rego code. Validation errors with specified lines coming back from rode align with what you see on the screen 🎉

### CODE REVIEW

File: components/CodeEditor.js [hunk 1]
```diff
@@ -35,6 +35,12 @@ const CodeEditor = (props) => {
   const { theme } = useTheme();
   const className = error ? styles.codeEditorError : styles.codeEditor;
 
+  const hightlightWithLineNumbers = (input) =>
```
Comment (alexashley) [inline components/CodeEditor.js:38]: Extra `t` in there
```suggestion
  const highlightWithLineNumbers = (input) =>
```

File: components/CodeEditor.js [hunk 2]
```diff
@@ -53,7 +59,8 @@ const CodeEditor = (props) => {
           tabSize={4}
           className={className}
           textareaClassName={className}
-          highlight={(code) => Prism.highlight(code, Prism.languages.rego)}
+          preClassName={styles.codeInEditor}
+          highlight={hightlightWithLineNumbers}
```
Comment (alexashley) [inline components/CodeEditor.js:63]:
```suggestion
          highlight={highlightWithLineNumbers}
```

File: prism/prism.js [hunk 3]
```diff
@@ -22,7 +22,7 @@
  SOFTWARE.
  */
 /* PrismJS 1.23.0
-https://prismjs.com/download.html#themes=prism-okaidia&languages=markup+css+clike+javascript+json+rego */
+https://prismjs.com/download.html#themes=prism-okaidia&languages=markup+css+clike+javascript+json+rego&plugins=line-numbers */
```
Comment (alexashley) [inline prism/prism.js:25]: Do we need the line number plugin at all? I removed it locally and looked like the numbering still worked correctly.

Comment (pickjasmine) [reply prism/prism.js:25]: I guess I did not try it without. Prism is supposed to support it out of the box but with the editor we're using it still required the manual mapping. Good catch!

**Review by alexashley — APPROVED** [review]
:shipit:

**Pull request merged**